audio/engine.asm

```diff
@@ -1763,10 +1709,12 @@ Music_SoundStatus:
 	set NOTE_PITCH_SWEEP, [hl]
 	ret
 
-Music_DutyCycle:
+Music_DutyCycle0:
+Music_DutyCycle1:
+Music_DutyCycle2:
+Music_DutyCycle3:
 ; duty cycle
-; params: 1
-	call GetMusicByte
+	ld a, [wCurMusicByte]
```

Comment:
If this depends on the numeric values of the music commands, there should be an `assert` to that effect.

Reply:
Not sure where best to do that, `assert duty_cycle_cmd & 3 == 0` in the macro file maybe?
Its arguably enforced by the jump table, octave does much the same and lacks an assert. the exact value isn't important, just the two lowest bits

Reply:
Good point about `octave`, it should have one too (in pokecrystal as well). In the command body is fine, like where there's currently a `; set octave based on lo nybble of the command` comment. Since shifting the values by 1 would break the implementation.

Reply:
Added asserts for duty_cycle and octave, plus sound_ret while i was there

Added them to the macro file because the _cmd defines are otherwise never used outside there and break the pattern of defines being all upper case